cli/tauri.js/src/helpers/rust-cli.ts

```diff
@@ -2,11 +2,16 @@ import { existsSync } from 'fs'
 import { resolve, join } from 'path'
 import { spawnSync, spawn } from './spawn'
 import { CargoManifest } from '../types/cargo'
+import { readTomlFile } from '../helpers/toml'
 
 const currentTauriCliVersion = (): string => {
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-var-requires, @typescript-eslint/no-unsafe-member-access
-  const tauriCliManifest = require('../../../core/Cargo.toml') as CargoManifest
-  return tauriCliManifest.package.version
+  const manifestPath = join(__dirname, '../../../core/Cargo.toml')
```

Comment:
the require() here is absolutely needed; with that, webpack automatically puts the right manifest in our bundle, so it works on the published version. if the require path is wrong, we should fix that instead.

Reply:
The `toml-loader` wasnt working. The path is fine.
You can try the `tauri init` with dev branch.

Maybe you can take a look because my branch is failing since I made a changes in `tauri.js` and this seems to fix it.

 Thanks

Reply:
Alos I tried with prefix `toml!.` i’ve been able to get it to work with some tweaking

https://www.npmjs.com/package/toml-loader

but did this already worked?